### ISSUE TITLE
Bug - IOS app - shutter sound continues when iphone storage full

### ISSUE THREAD

**Issue by geomapper69** [mapillary/mapillary_issues#551]
During a long sequence, when the storage becomes full, the message appears on the screen to warn the user.  However, the shutter sound will continue, even though nothing is being saved.
The shutter sound needs to stop (or change) when the storage becomes full.

**Comment by jesolem** [comment]
I'll take a look. Thanks.

**Comment by geomapper69** [comment]
Its intermittent.
Today the shutter sound stopped when the storage was full.
I will try to see if there is a pattern for when the bug occurs.

**Comment by geomapper69** [comment]
I have been doing som testing with Mapillary and other photo apps.  I found another app does strange things when storage becomes nearly full.  One time it stopped taking pictures saying the storage was full, but when I checked, it had 500 MB free.  Another time it kept taking pictures until the storage was completely full, but the last 300 or so pictures were completely out of focus (so the iphone camera had lost its ability to focus).
Mapillary has been behaving ok, but I have only filled the storage once since the original report.

So I will close this since I do not believe it is a Mapillary bug, just a limitation of IOS when the storage is full.
The only request I will make is to have a picture count limit, to prevent the storage becoming full.
